targets/psa_switch/psa_switch.cpp

```diff
@@ -316,8 +316,32 @@ PsaSwitch::ingress_thread() {
     Parser *parser = this->get_parser("ingress_parser");
     parser->parse(packet.get());
 
-    // set default std metadata values
+    // passing metadata
+    if (phv->has_field("psa_ingress_input_metadata.ingress_port")) {
```

Comment:
All of these has_field calls are guaranteed to return true, and can thus be removed.

Reply:
I think she removed them already